manifests/s/scarvalhojr/aoc-cli/0.4.6/scarvalhojr.aoc-cli.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: scarvalhojr.aoc-cli
+PackageVersion: 0.4.6
+PackageLocale: en-US
+Publisher: Sergio de Carvalho
+PackageName: Advent of Code command-line tool

```

```suggestion
PackageName: aoc-cli
```
This must match Control Panel.

---

🤦 

I'll fix it, thanks!!